v2/tools/generator/internal/codegen/golden_files_test.go

```diff
@@ -318,6 +325,36 @@ func stripUnusedTypesPipelineStage() *pipeline.Stage {
 		})
 }
 
+// stripFunctionsPipelineStage creates a stage to remove functions and interface implementations (which contain
+// functions) from the golden test output.
+// These golden tests are looking at the structure of the generated structs; including all the code for functions in
+// the output is just noise that makes the golden files hard to review.
+func stripFunctionsPipelineStage() *pipeline.Stage {
```

Comment:
We can't just pull those stages.

For one, they're often prerequisites of other stages that we do want to test.

For another, a lot of our stages are ONLY tested by these golden tests (they don't have isolated unit tests of their own). The more our test pipeline resembles the actual one we use for code generation the better, because we rule out unexpected interactions between the stages.

As discussed offline, I'll remove the filtering for now, and have logged #2150 so we remember to circle back and make things better.